src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/SubModel.h

```diff
@@ -73,47 +73,101 @@ class SubModel
     std::vector<std::string> m_externalContactList; /**< List of the additional external contacts */
 
 public:
+    /**
+     * Determines the validity of the object.
+     * @return True if the object is valid, false otherwise.
+     */
+    bool isValid() const;
+
     /**
      * Getters
      */
 
     /**
-     * @brief Access model.
+     * @brief Access m_model.
      * @return The model of the SubModel.
      */
     const iDynTree::Model& getModel() const;
 
     /**
-     * @brief Access jointListMapping.
+     * @brief Access m_jointListMapping.
      * @return the mapping between the joint indeces in the sub-model and the joint indeces in the
      * full-model.
      */
     const std::vector<int>& getJointMapping() const;
 
     /**
-     * @brief Access ftList.
+     * @brief Access m_ftList.
      * @return the list of FT objects which is the list of force/torque sensors.
      */
     const std::vector<FT>& getFTList() const;
 
     /**
-     * @brief Access accelerometerList.
+     * @brief Access m_accelerometerList.
      * @return a list of Sensor objects describing the accelerometers contained in the sub-model.
      */
     const std::vector<Sensor>& getAccelerometerList() const;
 
     /**
-     * @brief Access gyroscopeList.
+     * @brief Access m_gyroscopeList.
      * @return a list of Sensor objects describing the gyroscope contained in the sub-model.
      */
     const std::vector<Sensor>& getGyroscopeList() const;
 
     /**
-     * @brief Access externalContactList.
+     * @brief Access m_externalContactList.
      * @return a list of strings describing frame names of the external contacts for the sub-model.
      */
     const std::vector<std::string>& getExternalContactList() const;
 
+    /**
+     * @brief access the length of m_ftList.
+     * @return the number of force/torque sensors in the sub-model.
+     */
+    int getNrOfFTSensor() const;
```

Comment:
```suggestion
    std::size_t getNrOfFTSensor() const;
```

This is valid also foe the others

Reply:
Done